File: src/p1480/java/solution.java
class Solution {
    public int[] runningSum(int[] nums) {
        int[] sums = new int[nums.length];
        int sum = 0;
        for (int i = 0; i < nums.length; i++) {
            sum += nums[i];
            sums[i] = sum;
        }
        return sums;
    }
}

File: src/p1480/rust/solution.rs
impl Solution {
    pub fn running_sum(nums: Vec<i32>) -> Vec<i32> {
        let n = nums.len();
        let mut sums = vec![0; n];
        let mut sum = 0;
        for i in 0..n {
            sum += nums[i];
            sums[i] = sum;
        }
        return sums;
    }
}
